src/cli/onefuzz/templates/regression.py

```diff
@@ -0,0 +1,250 @@
+            target_env,
+            setup_dir,
+            reboot_after_setup,
+            target_options,
+            dryrun,
+            duration,
+            report_list,
+            crash_report_timeout,
+            debug,
+            check_retry_count,
+            check_fuzzer_help,
+            fail_on_repro,
+        )
+
+    def libfuzzer(
+        self,
+        project: str,
+        name: str,
+        build: str,
+        pool_name: str,
+        *,
+        crashes: Container = None,
+        input_reports: Container = None,
+        inputs: Optional[Directory] = None,
+        target_exe: File = File("fuzz.exe"),
+        tags: Optional[Dict[str, str]] = None,
+        notification_config: Optional[NotificationConfig] = None,
+        target_env: Optional[Dict[str, str]] = None,
+        setup_dir: Optional[Directory] = None,
+        reboot_after_setup: bool = False,
+        target_options: Optional[List[str]] = None,
+        dryrun: bool = False,
+        duration: int = 24,
+        report_list: Optional[List[str]] = None,
+        crash_report_timeout: Optional[int] = None,
+        debug: Optional[List[TaskDebugFlag]] = None,
+        check_retry_count: Optional[int] = None,
+        check_fuzzer_help: bool = True,
+        fail_on_repro: bool = False,
+    ) -> Optional[Job]:
+
+        return self._create_job(
+            TaskType.libfuzzer_regression,
+            project,
+            name,
+            build,
+            pool_name,
+            crashes,
+            input_reports,
+            inputs,
+            target_exe,
+            tags,
+            notification_config,
+            target_env,
+            setup_dir,
+            reboot_after_setup,
+            target_options,
+            dryrun,
+            duration,
+            report_list,
+            crash_report_timeout,
+            debug,
+            check_retry_count,
+            check_fuzzer_help,
+            fail_on_repro,
+        )
+
+    def _create_job(
+        self,
+        task_type: TaskType,
+        project: str,
+        name: str,
+        build: str,
+        pool_name: str,
+        crashes: Container = None,
+        input_reports: Container = None,
+        inputs: Optional[Directory] = None,
+        target_exe: File = File("fuzz.exe"),
+        tags: Optional[Dict[str, str]] = None,
+        notification_config: Optional[NotificationConfig] = None,
+        target_env: Optional[Dict[str, str]] = None,
+        setup_dir: Optional[Directory] = None,
+        reboot_after_setup: bool = False,
+        target_options: Optional[List[str]] = None,
+        dryrun: bool = False,
+        duration: int = 24,
+        report_list: Optional[List[str]] = None,
+        crash_report_timeout: Optional[int] = None,
+        debug: Optional[List[TaskDebugFlag]] = None,
+        check_retry_count: Optional[int] = None,
+        check_fuzzer_help: bool = True,
+        fail_on_repro: bool = False,
+    ) -> Optional[Job]:
+
```

Comment:
This would be extremely useful to add the pydoc documentation here, as `inputs` is conceptually very different than `crashes` + `input_reports`

see `AFL.basic` from `cli/onefuzz/templates/afl.py` for an example use to specify the arguments.